torchtune/utils/metric_logging.py

```diff
@@ -205,13 +210,14 @@ def log_config(self, config: DictConfig) -> None:
             config (DictConfig): config to log
         """
         if self._wandb.run:
```

Comment:
does this also prevent ranks other than rank 0 from logging the config?

Reply:
but is the fact that the run is active accessible from all ranks? will `if self._wandb.run` pass on rank 1 even though it was initialized on rank 0?